### PR TITLE
feat: add graphs throughout the site

### PR DESCRIPTION
**Pull request by johnletey**

### CODE REVIEW

File: src/components/app/Watchlist.svelte [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+    const returnedPrices = await client.price(token);
+    const length = returnedPrices.prices.length - 1;
+
+    const dates = returnedPrices.dates.slice(length - period, length);
+    const prices = returnedPrices.prices.slice(length - period, length);
+
+    const start = prices[0];
+    const end = prices[prices.length - 1];
+    let color;
+    let percentage = (end - start) / start;
```
Comment (t8) [inline src/components/app/Watchlist.svelte:97]: For percent change, I think you'll need to multiply this by 100 to get the actual percentage. I did the math to make sure this works, and it checks out.

Comment (johnletey) [reply src/components/app/Watchlist.svelte]: @t8 Yup 😅 That's absolutely correct. Fixed in https://github.com/useverto/verto/pull/91/commits/b34b5f2d6a60f4c576551995a7d1c8be91e3f61d.

**Review by t8 — CHANGES_REQUESTED** [review]
I **love** these additions. The only thing I think we should edit here is the hover state on the watchlist cards when they are in edit mode.
image

If we could remove the shadow around the graph itself (and leave the one around the X) that would be amazing. Again, incredible work @johnletey, @martonlederer, and @maximousblk (for the killer design)!

**Review by t8 — APPROVED** [review]
LGTM

This is **so** cool!

**Pull request merged**